confignode/src/main/java/org/apache/iotdb/confignode/service/ConfigNode.java

```diff
@@ -119,6 +119,27 @@ public void active() {
           "{} has registered successfully. Waiting for the leader's scheduling to join the cluster.",
           ConfigNodeConstant.GLOBAL_NAME);
 
+      boolean isJoinedCluster = false;
+      for (int retry = 0; retry < 20; retry++) {
```

Comment:
20 can be defined as a static final variable

Reply:
Fixed